cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -340,4 +340,20 @@ export class Assessment {
             "true"
         );
     }
+    public static validateReviewDonutChart(): void {
+        cy.get(".pf-v5-c-chart").should("be.visible");
+        cy.get(".pf-v5-c-chart svg").should("have.attr", "role", "img").and("be.visible");
+        cy.get('[id*="victory-container"]').should("contain", "risk-donut-chart");
```

Comment:
Hi Neil, Ids we can add in assessment.view

Reply:
done